src/h2olog/h2olog.cc

```diff
@@ -63,6 +64,9 @@ Optional arguments:
   -a                Include application data which are omitted by default.
   -r                Run without dropping root privilege.
   -w <path>         Path to write the output (default: stdout).
+  -f <flag>         Enables a BCC debug flag (supported flag: DEBUG_LLVM_IR,
```

Comment:
```suggestion
  -f <flag>         Turn on a BCC debug flag (supported flag: DEBUG_LLVM_IR,
```

We do not use third person singular verb elsewhere in the help message.